gradle.lockfile

```diff
@@ -0,0 +1,249 @@
+# This is a Gradle generated file for dependency locking.
+# Manual edits can break the build and are not advised.
```

Comment:
nit: maybe link to the readme, or copy the command here for the impatient?

Reply:
When Gradle regenerates the file, anything we added to it would be lost, so that's not really possible.